client/stylesheets/abstracts/_mixins.scss

```diff
@@ -43,9 +43,35 @@
 	}
 }
 
+// WooCommerce/Body Large
+@mixin wc-body-large {
+	font-size: 16px;
```

Comment:
Just curious: why are we not using the variables from `node_modules/@wordpress/base-styles/_variables.scss` for font sizes?
I mean `16px` -> `$grid-unit-20` as I saw above in this file
<img width="434" alt="image" src="https://user-images.githubusercontent.com/79862886/235100664-a756c0f3-ae4d-4772-a6f5-5d6e24bf80ec.png">

Reply:
Because those variables are for grid sizes (spacing). We lack a font size definition variables, that's why I've relayed on mixins instead. Although we should come up with a unified approach, I think this is better than just keep copy/pasting it every time 🙂

Reply:
Thanks for the explanation! 
Yep, agree with you that needs a unified approach, which is a topic for another discussion 🙂